server/routes/isUploaded.js

```diff
@@ -7,15 +7,16 @@ app.use(cors());
 app.use(bodyParser.json());
 // import { minioClient } from '../minioConfig.js';
 import { file_stats } from '../Database_queries/queries.js';
+import { logger, log } from '../logger.js';
```

Comment:
Ensure that the logger is properly configured in `../logger.js` to avoid runtime errors. If not already done, consider adding a check to confirm the logger is initialized correctly.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*